contract_payment_mode/__manifest__.py

```diff
@@ -1,15 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 Domatix (<www.domatix.com>)
 # Copyright 2016 Antiun Ingenieria S.L. - Antonio Espinosa
 # Copyright 2017 Tecnativa - Vicent Cubells
 # Copyright 2017 Tecnativa - David Vidal
 # Copyright 2017 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2018 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
```

Comment:
Don't duplicate lines, but move your line to the bottom and put the year intervals